Docker/install.sh

```diff
@@ -25,8 +25,8 @@ $RELEASE/common/cmspkg -a $SCRAM_ARCH upgrade
 $RELEASE/common/cmspkg -a $SCRAM_ARCH update
 $RELEASE/common/cmspkg -a $SCRAM_ARCH install cms+crabtaskworker+$RELEASE
 
-cp $MYTESTAREA/$SCRAM_ARCH/cms/crabtaskworker/$RELEASE/data/script/Deployment/Publisher/{start.sh,env.sh,stop.sh} /data/srv/Publisher/
-cp $MYTESTAREA/$SCRAM_ARCH/cms/crabtaskworker/$RELEASE/data/script/Deployment/TaskWorker/{start.sh,env.sh,stop.sh} /data/srv/TaskManager/
+cp $MYTESTAREA/$SCRAM_ARCH/cms/crabtaskworker/$RELEASE/data/script/Deployment/Publisher/{start.sh,env.sh,stop.sh,updateTMRuntime.sh} /data/srv/Publisher/
```

Comment:
hm, maybe I am missing something, but `updateTMRuntime.sh` is not available at `Deployment/Publisher/` directory. So this `cp` command should fail. OTOH, script is at `Deployment/TaskWorker/`, so we should take it from there.

Reply:
My bad. I made a new commit to rectify this up